spec/requests/api/visitor_can_view_articles_by_categories_spec.rb

```diff
@@ -0,0 +1,39 @@
+RSpec.describe 'GET/api/categories' do
+  let!(:category) { create(:category, label: 'global_politics') }
+  let!(:articles) { 2.times { create(:article, category: category) } }
+
+  describe 'successfully get articles sorted into categories' do
+    before do
+      get "/api/categories/#{category.label}"
+    end
+
+    it 'is expected to return a 200 response' do
+      expect(response).to have_http_status 200
+    end
+
+    it 'is expected to return all categories' do
+      expect(response_json['category'].count).to eq 3
```

Comment:
why would you get an array of 3 categories? shouldn't it be `response_json['category']['articles'].count` ? Make sure you are in control of the response object here